### PR TITLE
IGNITE-14564 Add LICENSE and NOTICE files to all artifacts.

### PR DESCRIPTION
**Pull request by ivandasch**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,13 @@
-recursive-include requirements *
+graft requirements
+graft examples
+graft docs
+recursive-exclude docs/generated *
+graft tests
```
Comment (isapego) [inline MANIFEST.in:5]: Why do we want tests in package?

Comment (ivandasch) [reply MANIFEST.in:5]: https://www.apache.org/legal/release-policy.html#source-packages
```
SOURCE PACKAGES
Every ASF release MUST contain one or more source packages, which MUST be sufficient for a user to build and test the release provided they have access to the appropriate platform and tools. A source release SHOULD not contain compiled code.
```

**Review by isapego — APPROVED** [review]
Ship it.

File: MANIFEST.in (hunk 1 above)
Comment (isapego) [reply MANIFEST.in:5]: Got it

**Pull request closed without merging**